packages/cli/src/lib/cmds/build.ts

```diff
@@ -25,6 +25,18 @@ import {printValidationResult} from '../pwaValidationHelper';
 import {ParsedArgs} from 'minimist';
 import {createValidateString} from '../inputHelpers';
 
+// Path to the file generated when building an app bundle file using gradle.
+const APP_BUNDLE_BUILD_OUTPUT_FILE_NAME = 'app/build/outputs/bundle/release/app-release.aab';
+const APP_BUNDLE_SIGNED_FILE_NAME = './app-release-bundle.aab'; // Final signed App Bundle file.
+
+// Path to the file generated when building an APK file using gradle.
+const APK_BUILD_OUTPUT_FILE_NAME = './app/build/outputs/apk/release/app-release-unsigned.apk';
```

Comment:
Is there any reason to leave this file around? I could see the user getting a little confused between the three files:

* a: `app-release-unsigned.apk`
* b: `app-release-unsigned-aligned.apk`
* c: `app-release-signed.apk`

I can see the users caring about whether or not the apk is signed, but I don't think anyone (other than us for development purposes) cares about the unaligned one (a).

Could we rename (a) to something else (`app-release-unaligned.apk`?) during the build process, and delete it afterwards. Then we can rename (b) to `app-release-unsigned.apk`, and then the user only sees:

* `app-release-unsigned.apk`
* `app-release-signed.apk`

Reply:
I wonder if there's anyone relying on those files as part of a CI process nowadays? This might be a breaking change for them.

Anyway, my goal with this PR was to avoid introducing any behaviour changes and just move code around.

Reply:
Fair enough - happy to not change things unnecessarily.